cmd/commands/image.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2018 The original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commands
+
+import (
+	"errors"
+	"github.com/projectriff/riff/pkg/core"
+	"github.com/spf13/cobra"
+)
+
+func Image() *cobra.Command {
+	return &cobra.Command{
+		Use:   "image",
+		Short: "Interact with docker images",
+	}
+}
+
+func ImageRelocate(c *core.Client) *cobra.Command {
+	options := core.RelocateImageOptions{}
+
+	command := &cobra.Command{
+		Use:   "relocate",
+		Short: "Relocate docker image names to another registry",
+		Long: `Relocate either a single kubernetes configuration file or a riff manifest and its kubernetes
+configuration files so that image names refer to another (private or public) registry.
+
+To relocate a single kubernetes configuration file, use the '--yaml' flag to specify the path or URL of the file. Use
+the '--output' flag to specify the path for the relocated file. If '--output' is an existing directory, the relocated
+file will be placed in that directory. Otherwise the relocated file will be written to the path specified in '--output'.
+
+To relocate a manifest, use the '--manifest' flag to specify the path of a manifest file which provides the paths or
+URLs of the kubernetes configuration files for riff components. Use the '--output' flag to specify the path of a
+directory to contain the relocated manifest and kubernetes configuration files.
+
+Specify the registry hostname using the '--registry' flag, the user owning the images using the '--registry-user' flag,
+and a complete list of the images to be mapped using the '--images' flag. The '--images' flag contains the path of an
+image manifest file with contents of the following form:
+
+    manifestVersion: 0.1
+    images:
+    ...
+    - istio/sidecar_injector
```

Comment:
From that example, wasn't clear to me at first that images should contain their tag/versions in there. Maybe should amend